src/apache/conf/httpd.conf

```diff
@@ -59,6 +60,31 @@ LoadModule dir_module modules/mod_dir.so
 LoadModule alias_module modules/mod_alias.so
 LoadModule rewrite_module modules/mod_rewrite.so
 LoadModule socache_shmcb_module modules/mod_socache_shmcb.so
+<IfDefine EnableHttp2>
+  LoadModule http2_module modules/mod_http2.so
+</IfDefine>
+
+
+
+<IfModule http2>
+  # Server Protocols
+  Protocols h2 h2c http/1.1
```

Comment:
I don't really think that we need **h2c** (HTTP/2 clear text) as the main browsers does not support it: https://www.mnot.net/blog/2015/06/15/http2_implementation_status

Reply:
I thought I'd leave it in since some non-browser clients might use it but since not even the nextcloud client uses it (https://github.com/owncloud/client/pull/5659/files#diff-b2c28f8a04cc15ec70196757a5a250ffR99), it's probably better to remove it.